frontend/app/.server/constants/types.constant.ts

```diff
@@ -135,6 +138,8 @@ export const TYPES = assignServiceIdentifiers({
       ApplicantDtoMapper: serviceId<ApplicantDtoMapper>(),
       ApplicationStatusDtoMapper: serviceId<ApplicationStatusDtoMapper>(),
       BenefitApplicationDtoMapper: serviceId<BenefitApplicationDtoMapper>(),
+      ApplicationYearDtoMapper: serviceId<ApplicationYearDtoMapper>(),
+      BenefitApplicationStateMapper: serviceId<BenefitApplicationStateMapper>(),
```

Comment:
This line that adds the `BenefitApplicationStateMapper` type can be removed as it is already in `routes.mappers` below.